sonar-scala-plugin/src/main/resources/org/sonar/l10n/scala/rules/scalastyle/scapegoat-rules.json

```diff
@@ -0,0 +1,835 @@
+[
+  {
+    "key": "AvoidToMinusOne",
+    "name": "Checks for loops that use `x to n-1` instead of `x until n`",
+    "type": "CODE_SMELL",
+    "severity": "MINOR",
+    "url": "https://github.com/sksamuel/scapegoat/blob/master/README.md"
```

Comment:
We should reference the table link: https://github.com/sksamuel/scapegoat/blob/master/README.md#inspections